src/adjoint/masterRoutines.F90

```diff
@@ -775,6 +846,61 @@ subroutine master_b(wbar, xbar, extraBar, forcesBar, dwBar, nState, famLists, &
              call updateWallDistancesQuickly_b(nn, 1, sps)
           end if
 
+          ! Here we insert the functions related to
+          ! rotational (mesh movement) setup
+          !if (myID == 0) then
+             !
+             !
+          if (myID == 0) then
+             print*,'R0-mham: entering t(:)-loop'
+          end if
+          ! mham: just give t(:) a zero initialization... we do not use it
+          ! t(:) = 0.d0
+          t = timeunsteadyrestart
+          if (equationmode .eq. timespectral) then
+             do mm=1,nsections
+                t(mm) = t(mm) + (sps-1)*sections(mm)%timeperiod/real(&
+                     &         ntimeintervalsspectral, realtype)
+             end do
+          end if
+          ! fake forward sweep
+          if (myID == 0) then
+             print*,'R0-mham: entering grid'
+          end if
+          call gridvelocitiesfinelevel_block(useoldcoor, t, sps)
+          if (myID == 0) then
+             print*,'R0-mham: entering normal'
+          endif
+          call normalvelocities_block(sps)
+          if (myID == 0) then
+             print*,'R0-mham: entering slip'
+          end if
+          call slipvelocitiesfinelevel_block(useoldcoor, t, sps)
+          ! required for ts
+          if (myID == 0) then
+             print*,'R0-mham: entering slip_b'
+          end if
+          call slipvelocitiesfinelevel_block_b(useoldcoor, t, sps)
+          if (myID == 0) then
+             print*,'R0-mham: entering normal_b'
+          end if
+          ! required for ts
+          call normalvelocities_block_b(sps)
+          ! mham: normalvelocities_block_b zeros the faceid.
+          !       this is probably a bad thing
```

Comment:
these are zeroed because we use the pointer stuff in these routines. these variables getting zeroed out is okay, because it happens after we copy out the useful stuff

So feel free to remove this comment, and the similar one below